docs/getting_started.md

```diff
@@ -15,6 +15,9 @@
 5. For UI testing follow the instructions in [openshift console](https://github.com/openshift/console#dependencies) repository.
    After clone of repository and setting all dependencies please create configuration file with RUN section with `openshift_console_path`
    parameter pointing to your local clone of openshift console repo. See example in `conf/ocsci/ui-testing.yaml`.
+6. Install the dependencies for scipy
```

Comment:
I'm on OSX so I haven't had to do this during setup. @vasukulkarni can you confirm that this is necessary for users when setting up on linux? I assume it is but I just want to verify.

Reply:
successfully built on fedora 28 using 
```
$ cat /etc/redhat-release 
Fedora release 28 (Twenty Eight)

$ rpm -qa | grep 'atlas\|libffi\|openssl\|gcc'
openssl-1.1.0i-1.fc28.x86_64
libffi-3.1-16.fc28.x86_64
openssl-pkcs11-0.4.8-2.fc28.x86_64
libgcc-8.3.1-2.fc28.x86_64
openssl-libs-1.1.0i-1.fc28.x86_64
```

Reply:
@vasukulkarni were you able to complete the change?  @manojnkumar did you see the question ^^

Reply:
@vasukulkarni is it complete?  Should this one be closed?